package.json

```diff
@@ -25,7 +25,7 @@
     "gpu.js": "^2.10.4",
     "lodash": "^4.17.20",
     "node-getopt": "^0.3.2",
-    "source-map": "^0.7.3",
+    "source-map": "0.7.3",
```

Comment:
Pin `source-map` version for now due to breaking change in new minor patch

Ref: https://github.com/mozilla/source-map/pull/350

It changes the way `source-map` detects execution environment. New change will require refactor to how we handle tests `js-slang`.

> Previously, it uses the presence of `fetch` api to determine if it is a browser env. After the change it checks for the  presence`window` object. Since we run tests in a `node` env, tests that depends on source map (typically those that checks for error messages in code snippets) will fail.
We were able to pass tests previously because `source-map` interprets our testing environment as non-browser since it checks for `fetch`.